### ISSUE TITLE
Artist name with non-Latin characters

### ISSUE THREAD

**Issue by AndreyDyakonov** [BeamMW/nft-marketplace#93]
If the user uses non-Latin characters when creating an Artist (in the artist's name or in the description), then the name or description is not displayed correctly

**When a moderator rejects an Artist, the user has no way to change the artist's name to the correct one or create a new Artist.** 

some solution is needed, for example, to prohibit the input of characters other than the Latin alphabet, or to allow the user to correct the name after rejection by the moderator

<img width="550" src="https://user-images.githubusercontent.com/85252690/174976848-1e302235-5760-4f44-a486-ed21a61bbaf9.jpg">

<img width="550" src="https://user-images.githubusercontent.com/85252690/174973841-996c0bb3-6fb0-4a10-9d9d-a1ad04902e15.jpg">

<img width="550" src="https://user-images.githubusercontent.com/85252690/174973671-ec4bac6c-e38f-417c-9df3-3e063e9f1726.jpg">

**Comment by chapati** [comment]
Non-latin text is disabled for now. Please test and move to backlog, should enable in the next version

**Comment by AndreyDyakonov** [comment]
Non-latin text is disabled for now, but non-Latin text can still be added to  Artist's name or description via copy/paste
In new  version it will be necessary to prohibit this too

https://user-images.githubusercontent.com/85252690/175921595-336465fc-03d5-48fc-b93a-ad0a23f95b6b.mp4

move this to backlog
